src/main/java/org/folio/edge/core/EdgeVerticle2.java

```diff
@@ -87,38 +84,15 @@ public void start(Future<Void> future) {
 
     final Router router = defineRoutes();
 
-    server.requestHandler(router::accept).listen(port, result -> {
-      if (result.succeeded()) {
-        future.complete();
-      } else {
-        future.fail(result.cause());
-      }
-    });
+    server.requestHandler(router).listen(port)
```

Comment:
server.requestHandler(router).listen(port).onComplete(promise);

Reply:
Same here. That construct does not compile.

Reply:
```
    server.requestHandler(router)
    .listen(port)
    .<Void>mapEmpty()
    .onComplete(promise);
```

Reply:
ok